tests/test_amity.py

```diff
@@ -0,0 +1,215 @@
+import unittest
+from app.amity import Amity
+from app.person import Fellow, Staff
+
+
+class TestAmity(unittest.TestCase):
+    def setUp(self):
+        self.amity = Amity()
+
+    def test_create_room(self):
+        '''test to confirm a room has been created'''
+        rooms = self.amity.all_rooms
+        # making sure the rooms are empty
+        self.assertEqual(len(rooms), 0)
+        self.amity.create_room('haskel', 'livingspace')
+        # checking to see if rooms have been added
+        self.assertEqual(len(rooms), 1)
```

Comment:
maybe you can test for an increment of 1 instead of testing for `0` and `1` themselves